programs/drift/src/controller/orders.rs

```diff
@@ -309,7 +309,7 @@ pub fn place_perp_order(
         )?;
     }
 
-    if force_reduce_only && risk_increasing {
+    if force_reduce_only && !user.orders[new_order_index].reduce_only {
```

Comment:
so if a market is in reduce only mode, a user's order will fail without explicitly setting reduce only flag?

Reply:
feel like a method `is_reducing()` would be nicer?

Reply:
is_reducing on what? above if the force_reduce_only is true, the order param is forced to be true